### PR TITLE
Transaction service - style updates

### PR DESCRIPTION
**Pull request by tanay1337**

### CODE REVIEW

File: safe-core-api/safe-transaction-service.md [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
+Safe Transaction Service keeps track of transactions sent via Safe contracts. It indexes these transactions using events (L2 chains) and tracing (L1 chains) mechanisms.
 
 **Key Features:**  
 
-- [**Blockchain Indexing**](#blockchain-indexing): Executed transactions, configuration changes, ERC-20/721 transfers, and onchain confirmations are automatically indexed from the blockchain.
-- [**Offchain transaction signatures**](#offchain-transaction-signatures): Transactions can be sent to the service, enabling offchain signature collection. This feature helps inform owners about pending transactions that are awaiting confirmation to be executed.
+- [**Blockchain indexing**](#blockchain-indexing): Executed transactions, configuration changes, ERC-20/721 transfers, and onchain confirmations are automatically indexed from the blockchain.
+- [**Offchain transaction signatures**](#offchain-transaction-signatures): Transactions are sent to the service, enabling offchain signature collection. This feature helps inform owners about pending transactions awaiting confirmation for execution.
```
Comment (germartinez) [inline safe-core-api/safe-transaction-service.md:8]: Off-chain signatures are not mandatory, but a very conveninent feature that is optional (used by default in our Web app).
```suggestion
- [**Offchain transaction signatures**](#offchain-transaction-signatures): Transactions can be sent to the service, enabling offchain signature collection. This feature helps inform owners about pending transactions awaiting confirmation for execution.
```

**Comment by germartinez** [comment]
Verbs describing endpoins should be in 3rd person singular.
This:
```- `GET /v1/safes/{address}/messages/`: Returns the messages created for the given Safe address.```
Instead:
```- `GET /v1/safes/{address}/messages/`: Return the messages created for the given Safe address.```

**Comment by tanay1337** [comment]
> Verbs describing endpoins should be in 3rd person singular. This: `` - `GET /v1/safes/{address}/messages/`: Returns the messages created for the given Safe address. `` Instead: `` - `GET /v1/safes/{address}/messages/`: Return the messages created for the given Safe address. ``

Do you see an instance where this rule isn't followed? Couldn't find anything. @germartinez

**Comment by germartinez** [comment]
Lines 69-70 and 111-114.

**Comment by tanay1337** [comment]
@germartinez Looks correct to me!
Screenshot 2023-10-16 at 12 46 28 PM
Screenshot 2023-10-16 at 12 45 48 PM

**Comment by germartinez** [comment]
I mean using "returns" instead of "return"

**Review by germartinez — APPROVED** [review]

**Pull request merged**